packages/nuxt/playground/app.vue

```diff
@@ -6,6 +6,7 @@ const counter = useCounter()
 useTestStore()
 useSomeStoreStore()
 
+const layerStore = useLayerStore()
```

Comment:
I think we don't need to change the playground code. We can treat the current playground as a layer. Refer to #2828.
What do you think ?@posva

Reply:
Why not. I will need to take a proper look next time